File: OmkarFCUBEAPI/FleetTrans/Models/BillStatementSearchModel.cs
namespace FleetTrans.Models
{
    public class BillStatementSearchModel
    {
        public string ConsignmentID { get; set; }
        public string BookedAt { get; set; }
        public string GcNoteNo { get; set; }
        public string BookingDate { get; set; }
        public string VehicleNo { get; set; }
        public string ProductName { get; set; }
        public string NoPackages { get; set; }
        public string Kms { get; set; }
        public string Rate { get; set; }
        public string GtotalRs { get; set; }
        public bool Selected { get; set; }
    }
}

File: OmkarFCUBEAPP/src/app/models/billstatementsearchmodel.ts

import { Billstatementsearchlistmodel } from "./billstatementsearchlistmodel";

export class Billstatementsearchmodel {

    consignmentID: string = "";
    bookedAt: string = "";
    gcNoteNo: string = "";
    bookingDate: string = "";
    vehicleNo: string = "";
    productName: string = "";
    noPackages: string = "";
    gtotalRs: string = "";
    selected: boolean = true;
    billStatementListData: Billstatementsearchmodel[]=[];
}
